### PR TITLE
Fix histogram widget filter

### PR DESCRIPTION
**Pull request by elenatorro**
Use categorical or numeric histogram depending on the property type instead of the buckets

Related issues:
* https://github.com/CartoDB/cartoframes/issues/929
* https://github.com/CartoDB/cartoframes/issues/940

histogram_filter

### CODE REVIEW

File: cartoframes/assets/src/widgets.js [hunk 1]
```diff
@@ -2,18 +2,26 @@ import { format } from './utils';
...
     widget.element.innerText = typeof value === 'number' ? format(value) : value;
   }
 }
 
-export function renderBridge(bridge, widget) {
+export function renderBridge(bridge, widget, mapLayer) {
   widget.element = widget.element || document.querySelector(`#${widget.id}`);
+  const type = mapLayer.metadata.properties[widget.value].type;
+
+  console.log('!!! renderBridge', widget);
```
Comment (rjimenezda) [inline cartoframes/assets/src/widgets.js]: :eyes:

File: cartoframes/assets/src/widgets.js [hunk 2]
```diff
@@ -2,18 +2,26 @@ import { format } from './utils';
...
 
   switch (widget.type) {
     case 'histogram':
-      bridge.histogram(widget.element, widget.value, widget.options);
+      if (type === 'category') {
```
Comment (rjimenezda) [inline cartoframes/assets/src/widgets.js]: This is way better. I really don't like the implicit 'histogram' that decides for you, and it was my idea  sweat_smile

File: cartoframes/assets/src/widgets.js (hunk 1 above)
Comment (elenatorro) [reply cartoframes/assets/src/widgets.js]: 😬

File: cartoframes/assets/src/widgets.js (hunk 2 above)
Comment (elenatorro) [reply cartoframes/assets/src/widgets.js]: I know 👌 , I'd like to change the `buckets` check in Airship at some point, but right now this is great to make it work in CARTOframes without releasing a new version for Airship.

Comment (rjimenezda) [reply cartoframes/assets/src/widgets.js]: I'd leave that bridge.histogram for compatibility's sake, but I would never use it personally :man_shrugging:

**Pull request merged**